llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -664,6 +672,15 @@ void LoadInst::dump() const {
   dbgs() << "\n";
 }
 #endif // NDEBUG
+
+void StoreInst::setVolatile(bool V) {
+  auto &Tracker = Ctx.getTracker();
+  if (Tracker.isTracking())
+    Tracker.track(
+        std::make_unique<SetVolatile>(cast<Instruction>(this), Tracker));
```

Comment:
same here: `cast<Instruction>(this)` -> `this`.

Reply:
Yeah, that's more sensible.